release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -467,7 +467,32 @@ In the table, features are marked with the following statuses:
+
+* Previously, the Developer Console did not allow images from insecure registries to be imported. This bug fix adds a checkbox that allows users to use the insecure registries in the *Deploy image* form. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826740[BZ#1826740])
+
+* When a user selected the *From Catalog* option to create an application, the *Developer Catalog* displayed a blank page instead of a list of templates to create an application. This was caused when the 1.18.0 Jaeger Operator was installed. This issue has now been fixed and the templates are displayed as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1845279[BZ#1845279])
+
+* When deleting a parallel task in a Pipeline through the *Pipeline Builder* in the Developer Console, the interface was rearranging the tasks connected to the parallel task incorrectly, creating orphan tasks. With this fix, the tasks connected to the deleted parallel task are reconnected with the original Pipeline. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856155[BZ#1856155])
+
+* The web console was crashing with a JavaScript exception when the user cancelled the creation of a Pipeline through the web console with a side panel opened at the same time. This was fixed by improving the internal state handling. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856267[BZ#1856267])
+
+* A user with the required permissions was unable to retrieve and deploy an image from another project. The required RoleBindings have now been created to fix this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843222[BZ#1843222])
+
+* When you tried to deploy an application from a Git repository with the *Import from Git* function, the Developer Console reported a false positive error `Git repository is not reachable` for private repositories reachable by the cluster. This was fixed by adding information on making the private repository available to the cluster in the error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1877739[BZ#1877739])
+
+* When a Go application was created through the Developer Console, a route to the application was not created. This was caused by a bug in `build-tools` and incorrectly configured ports. The issue has been fixed by picking either the user-provided port or the default port 8080 as the target port. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874817[BZ#1874817])
+
+* When you created an application with the *Import from Git* function, a subsequent change of the application's Git repository from the web console was not possible. This was caused by changing the application name in subsequent editing of the Git repository URL. This was fixed with making the application name read-only when editing the application Git repository URL. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1873095[BZ#1873095])
```

Comment:
This was caused by changing the application name in subsequent editing of the Git repository URL./ This was because the application name also changed when the Git repository URL was modified.

with making the application name/by making the application name

Reply:
leaving original + applying second suggestion
```suggestion
* When you created an application with the *Import from Git* function, a subsequent change of the application's Git repository from the web console was not possible. This was caused by changing the application name in subsequent editing of the Git repository URL. This was fixed by making the application name read-only when editing the application Git repository URL. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1873095[BZ#1873095])
```